bigquery/system-test/tables.test.js

```diff
@@ -63,20 +63,17 @@ describe('bigquery:tables', function () {
   });
 
   after(function (done) {
-    // Delete srcDataset
-    bigquery.dataset(srcDataset).delete({ force: true }, function () {
-      // Delete destDataset
-      bigquery.dataset(destDataset).delete({ force: true }, function () {
-        // Delete files
-        storage.bucket(options.bucket).deleteFiles({ force: true }, function (err) {
-          if (err) {
-            return done(err);
-          }
-          // Delete bucket
-          setTimeout(function () {
-            storage.bucket(options.bucket).delete(done);
-          }, 2000);
-        });
+    // Delete testing dataset/table
+    bigquery.dataset(options.dataset).delete({ force: true }, function () {
+      // Delete files
+      storage.bucket(options.bucket).deleteFiles({ force: true }, function (err) {
+        if (err) {
+          return done(err);
+        }
+        // Delete bucket
+        setTimeout(function () {
+          storage.bucket(options.bucket).delete(done);
+        }, 2000);
```

Comment:
Why these changes? Don't we still need to delete both `srcDataset` _and_ `destDataset`?